cmd/clusterctl/client/config/variables_client.go

```diff
@@ -19,6 +19,8 @@ package config
 const (
 	// GitHubTokenVariable defines a variable hosting the GitHub access token.
 	GitHubTokenVariable = "github-token"
+	// GitLabAccessTokenVariable defines a variable hosting the GitHub access token. This can be used with Personal and Project access tokens.
```

Comment:
nit
```suggestion
	// GitLabAccessTokenVariable defines a variable hosting the GitLab access token. This can be used with Personal and Project access tokens.
```